sw/host/ot_certs/src/x509/extension.rs

```diff
@@ -228,6 +228,31 @@ impl<'a> asn1::SimpleAsn1Readable<'a> for DiceTcbInfoFlags {
     }
 }
 
+impl<'a> asn1::SimpleAsn1Readable<'a> for KeyUsage {
+    const TAG: asn1::Tag = asn1::OwnedBitString::TAG;
+    fn parse_data(_data: &'a [u8]) -> asn1::ParseResult<Self> {
+        let result = asn1::OwnedBitString::parse_data(_data)?;
+        let bs = result.as_bitstring();
+        // List of bits that this parser supports. Any bit set outside of
+        // these will be an error because we cannot record it.
+        const PARSED_BITS: &[usize] = &[0, 4, 5];
```

Comment:
where do these bit values come from: 

This site does not seem in agreement: https://learn.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509keyusageflags?view=net-8.0

Reply:
I believe they come from: https://www.rfc-editor.org/rfc/rfc5280.html#section-4.2.1.3

Maybe we should link the src in a comment?

Reply:
which agrees with another source here: https://docs.redhat.com/en/documentation/red_hat_certificate_system/9/html/administration_guide/standard_x.509_v3_certificate_extensions#Standard_X.509_v3_Certificate_Extensions-keyUsage

Reply:
All those bits values are documented in the `push_key_usage_ext` function, I didn't want to duplicate the documentation there but maybe I'll just copy-paste it anyways. In any case, the bits in PARSED_BITS are simply the bits that the code right below handles. This is to make sure that the parse will fail if for any reason there is a bit set that cannot be represented by the KeyUsage structure.

Alternatively, we could handle all possibles bits, even if they are very unlikely to be used.

Reply:
Maybe mention in the comments that bit values come from RFC5280?